### DISCUSSION TITLE
SSD Nodes

### DISCUSSION THREAD

**Discussion by nnn19** [VarunAgw/CloudPingTest#4] [Ideas]
Hi

Simple and good job.

Can you please add SSD Nodes
US - New York, Atlanta, Dallas, Chicago, Los Angeles, Seattle
Germany - Frankfurt
Netherlands - Amsterdam
UK - London
IN - Mumbai
SG - Singapore
AU - Sydney
Japan - Tokyo

Thanks

Natarajan N Napoleon

**Comment by VarunAgw** [comment]
https://www.ssdnodes.com/ for reference

**Comment by nnn19** [comment]
SSDNodes have added few more Locations.
